### ISSUE TITLE
3.0.0 : Crash when opening Sequence

### ISSUE THREAD

**Issue by Louspirit** [fpwong/BlueprintAssistWiki#36]
Hello,

I encounter a crash when trying to open a Sequence after updating Blueprint Assist to 3.0.0 on UE4.26.2.

Disabling Blueprint Assist allows the editor to not crash.

Here is the log of the crash:

```
Assertion failed: IsValid() [File:D:\RocketSync\4.26.0-14830424+++UE4+Release-4.26\Working\Engine\Source\Runtime\Core\Public\Templates/SharedPointer.h] [Line: 890] 



0x00007ff99d094b89 KERNELBASE.dll!UnknownFunction []
0x00007ff90c6ed066 UE4Editor-Core.dll!UnknownFunction []
0x00007ff90c6f07a8 UE4Editor-Core.dll!UnknownFunction []
0x00007ff90c3fe8cd UE4Editor-Core.dll!UnknownFunction []
0x00007ff90c395575 UE4Editor-Core.dll!UnknownFunction []
0x00007ff90c397620 UE4Editor-Core.dll!UnknownFunction []
0x00007ff8f242e433 UE4Editor-BlueprintAssist.dll!FBAAssetEditorHandler::OnAssetOpenedInEditor() [D:\Build\++Portal\Sync\LocalBuilds\PluginTemp\HostProject\Plugins\BlueprintAssist\Source\BlueprintAssist\Private\BlueprintAssistAssetEditorHandler.cpp:120]
0x00007ff8f23b119e UE4Editor-BlueprintAssist.dll!TBaseRawMethodDelegateInstance<0,FBAAssetEditorHandler,void __cdecl(UObject *,IAssetEditorInstance *),FDefaultDelegateUserPolicy>::ExecuteIfSafe() [D:\RocketSync\4.26.0-14830424+++UE4+Release-4.26\Working\Engine\Source\Runtime\Core\Public\Delegates\DelegateInstancesImpl.h:469]
0x00007ff9071adf13 UE4Editor-UnrealEd.dll!UnknownFunction []
0x00007ff9071de9b6 UE4Editor-UnrealEd.dll!UnknownFunction []
0x00007ff9071dea92 UE4Editor-UnrealEd.dll!UnknownFunction []
0x00007ff90727c15a UE4Editor-UnrealEd.dll!UnknownFunction []
0x00007ff90727c2d8 UE4Editor-UnrealEd.dll!UnknownFunction []
0x00007ff8ecb43f30 UE4Editor-LevelSequenceEditor.dll!UnknownFunction []
0x00007ff8ecb4f77b UE4Editor-LevelSequenceEditor.dll!UnknownFunction []
0x00007ff9071e2c11 UE4Editor-UnrealEd.dll!UnknownFunction []
0x00007ff9071e3fad UE4Editor-UnrealEd.dll!UnknownFunction []
0x00007ff9071e3cda UE4Editor-UnrealEd.dll!UnknownFunction []
0x00007ff8f14c9fdf UE4Editor-ContentBrowserAssetDataSource.dll!UnknownFunction []
0x00007ff8f14ca200 UE4Editor-ContentBrowserAssetDataSource.dll!UnknownFunction []
0x00007ff8f14b7c82 UE4Editor-ContentBrowserAssetDataSource.dll!UnknownFunction []
0x00007ff8fc82b95d UE4Editor-ContentBrowser.dll!UnknownFunction []
0x00007ff8fc7ea800 UE4Editor-ContentBrowser.dll!UnknownFunction []
0x00007ff8fc74b74c UE4Editor-ContentBrowser.dll!UnknownFunction []
0x00007ff8fc709357 UE4Editor-ContentBrowser.dll!UnknownFunction []
0x00007ff8fc668374 UE4Editor-ContentBrowser.dll!UnknownFunction []
0x00007ff8fc756227 UE4Editor-ContentBrowser.dll!UnknownFunction []
0x00007ff8fc74b93b UE4Editor-ContentBrowser.dll!UnknownFunction []
0x00007ff9081b2bf8 UE4Editor-Slate.dll!UnknownFunction []
0x00007ff9082368e7 UE4Editor-Slate.dll!UnknownFunction []
0x00007ff90821f4ce UE4Editor-Slate.dll!UnknownFunction []
0x00007ff908212de9 UE4Editor-Slate.dll!UnknownFunction []
0x00007ff92510bc8f UE4Editor-ApplicationCore.dll!UnknownFunction []
0x00007ff9250f9197 UE4Editor-ApplicationCore.dll!UnknownFunction []
0x00007ff92510e389 UE4Editor-ApplicationCore.dll!UnknownFunction []
0x00007ff9250f4310 UE4Editor-ApplicationCore.dll!UnknownFunction []
0x00007ff99e55e858 USER32.dll!UnknownFunction []
0x00007ff99e55e299 USER32.dll!UnknownFunction []
0x00007ff92510f914 UE4Editor-ApplicationCore.dll!UnknownFunction []
0x00007ff7d0c58848 UE4Editor.exe!UnknownFunction []
0x00007ff7d0c7117c UE4Editor.exe!UnknownFunction []
0x00007ff7d0c7125a UE4Editor.exe!UnknownFunction []
0x00007ff7d0c852bd UE4Editor.exe!UnknownFunction []
0x00007ff7d0c87fea UE4Editor.exe!UnknownFunction []
0x00007ff99e247034 KERNEL32.DLL!UnknownFunction []
0x00007ff99f522651 ntdll.dll!UnknownFunction []

```

Tried disabling _Treat Delegates as Execution Pins_  and _Auto Position Event nodes_ with no luck.

**Comment by fpwong** [comment]
Hi Louspirit, thanks for the bug report. 

I can confirm this crash occurs on my side, will look into fixing this.

**Comment by fpwong** [comment]
This issue has been fixed in the latest update (3.0.1).

**Comment by Louspirit** [comment]
Fix confirmed. Thank you for your reactivity.
